### PR TITLE
Merge mvp_demo to master

### PR DESCRIPTION
**Pull request by dinogun**
Content includes releases [0.0.9_mvp ](https://github.com/kruize/autotune/milestone/16?closed=1) and [0.0.10_mvp](https://github.com/kruize/autotune/milestone/17?closed=1)

### CODE REVIEW

**Review by chandrams — APPROVED** [review]
LGTM!

File: src/main/java/com/autotune/common/data/result/RecommendationNotification.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.autotune.common.data.result;
```
Comment (bharathappali) [inline src/main/java/com/autotune/common/data/result/RecommendationNotification.java:1]: LICENSE header missing. Added in #631 

#631 needs to be merged before #627 re-review

**Review by bharathappali — COMMENTED** [review]
Everything looks good except the missing LICENSE header in `RecommendationNotification.java`.

**Review by bharathappali — APPROVED** [review]
LGTM!

**Pull request merged**